templates/data/postgresql/support.html

```diff
@@ -0,0 +1,635 @@
+            </li>
+            <li class="p-list__item has-bullet">
+              Easily add extensions to your PostgreSQL installations.
+            </li>
+            <li class="p-list__item has-bullet">
+              Easily manage multiple PostgreSQL instances in the same node using pg_* utilities such as <a href="https://manpages.ubuntu.com/manpages/oracular/en/man1/pg_lsclusters.1.html">pg_lsclusters</a>.
+            </li>
+          </ul>
+        </div>  
+      </div>
+      <hr class="p-rule--muted"/>
+      <div class="row">
+        <div class="col-3">
+          <h5>Snaps</h5>
+        </div>
+        <div class="col-6">
+          <p>
+            Get snaps for added confinement and atomic updates on all major <a href="https://snapcraft.io/docs/distro-support">Linux distributions</a>. We support two options:
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet">
+              PostgreSQL-only snap for the smallest possible footprint and attack surface.
+            </li>
+            <li class="p-list__item has-bullet">
+              Charmed PostgreSQL snap for a complete solution integrating the most popular PostgreSQL extensions, backup, monitoring and high availability tools.
+            </li>
+          </ul>
+        </div>  
+      </div>
+      <hr class="p-rule--muted"/>
+      <div class="row">
+        <div class="col-3">
+          <h5>Containers</h5>
+        </div>
+        <div class="col-6">
+          <p>
+            We deliver OCI-compliant images for your containerized environment. We provide several supported options:
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet">
+              PostgreSQL-only regular <a href="https://hub.docker.com/r/ubuntu/postgres">container</a>.
+            </li>
+            <li class="p-list__item has-bullet">
+              PostgreSQL-only <a href="https://documentation.ubuntu.com/rockcraft/en/latest/explanation/chisel/#chisel-explanation">chiselled</a> rock for the smallest possible footprint and attack surface.
+            </li>
+            <li class="p-list__item has-bullet">
+              Charmed PostgreSQL <a href="https://documentation.ubuntu.com/rockcraft/en/latest/explanation/rocks/">rock</a> for a complete solution integrating the most popular PostgreSQL extensions, backup, monitoring and high availability tools.
+            </li>
+          </ul>
+        </div>  
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="row--50-50">
+    <hr class="p-rule"/>
+    <div class="col">
+      <h2>PostgreSQL consulting</h2>
+      <div class="p-image-container--16-9 is-highlighted">
+        <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/7bf5c40d-Canonical%20OS%20Infrastructure%20Illustrations%20FINAL_Hardware%20Ecosystem.svg" width="600" alt="Canonical hardware ecosystem"/>
```

Comment:
The PostgreSQL consulting image is currently not using the image template due to an error I am encountering with the image template. 

See [MM](https://chat.canonical.com/canonical/pl/f7uox7cf3bno7y8wxmjybr7naw) for details

Reply:
Fixed